shared/chat/conversation/messages/unfurl/prompt-list/container.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import * as Constants from '../../../../../constants/chat2'
+import * as Chat2Gen from '../../../../../actions/chat2-gen'
+import * as Types from '../../../../../constants/types/chat2'
+import * as RPCChatTypes from '../../../../../constants/types/rpc-chat-gen'
+import * as I from 'immutable'
+import {namedConnect} from '../../../../../util/container'
+import UnfurlPromptList from '.'
+
+type OwnProps = {|conversationIDKey: Types.ConversationIDKey, ordinal: Types.Ordinal|}
+
+const noPrompts = I.Set()
+const noMessageID = Types.numberToMessageID(0)
+
+const mapStateToProps = (state, {conversationIDKey, ordinal}: OwnProps) => {
+  const message = Constants.getMessage(state, conversationIDKey, ordinal)
+  return {
+    promptDomains: message && message.type === 'text' ? message.unfurlPrompts : noPrompts,
+    messageID: message && message.type === 'text' ? message.id : noMessageID,
+  }
+}
+
+const mapDispatchToProps = (dispatch, {conversationIDKey}: OwnProps) => {
+  return {
+    _setPolicy: (messageID: Types.MessageID, domain: string, result: RPCChatTypes.UnfurlPromptResult) => {
+      dispatch(
+        Chat2Gen.createUnfurlResolvePrompt({
+          conversationIDKey,
+          messageID,
+          domain,
+          result,
+        })
+      )
+    },
+  }
+}
+
+const makeRes = (actionType: any, domain?: string) => {
```

Comment:
why do you need this?

Reply:
I couldn't figure out how to get Flow to pass without taking `actionType` to `any`. I don't understand the error it gives back so I went with this, which I thought looks ok anyway.

Reply:
If you could fix that would be great